Detectors/EMCAL/reconstruction/include/EMCALReconstruction/CTFCoder.h

```diff
@@ -169,8 +206,16 @@ o2::ctf::CTFIOSize CTFCoder::decode(const CTF::base& ec, VTRG& trigVec, VCELL& c
   uint32_t firstEntry = 0, cellCount = 0;
   o2::InteractionRecord ir(header.firstBC, header.firstOrbit);
 
+  CellCompressed cellCompressed;
   Cell cell;
   TriggerRecord trg;
+
+  // perform a check on dict we have old CTF with different energy compression
+  Bool_t isVersion1 = kFALSE;
+  if (header.majorVersion == 0 && header.minorVersion == 1) {
```

Comment:
we need to validate if this check is correct, this is my guess how it should be done